samples/isilon_v270_k8s_124.yaml

```diff
@@ -5,15 +5,15 @@ metadata:
   namespace: test-isilon
 spec:
   driver:
-    # Config version for CSI PowerScale v2.4.0 driver
-    configVersion: v2.4.0
+    # Config version for CSI PowerScale v2.6.0 driver
+    configVersion: v2.6.0
     replicas: 2
     dnsPolicy: ClusterFirstWithHostNet
     forceUpdate: false
     fsGroupPolicy: ReadWriteOnceWithFSType
     common:
-      # Image for CSI PowerScale driver v2.4.0
-      image: "dellemc/csi-isilon:v2.4.0"
+      # Image for CSI PowerScale driver v2.6.0
+      image: "dellemc/csi-isilon:v2.6.0"
```

Comment:
Please change the image and the corresponding comment to point to 2.7.0. Same comment for all the K8s samples below